scd/scd.h

```diff
@@ -26,13 +26,21 @@
 
 #include "softtoken.h"
 
+#define PROVISIONING_MODE_FILE "/tmp/_provisioning_"
```

Comment:
my slight preference would be to check whether the device is in provisioning mode based on the existence of the softtoken, device cert, and sw sign trust anchors. in case provisioning somehow fails or is not completed and the device is rebooted, the virtualization layer locks provisioning mode with this file created.

Reply:
leave this one open for future commit